src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,17 @@
+.cards {
+  display: flex;
+  flex-wrap: wrap;
+  box-sizing: border-box;
+  padding: 50px 40px;
+  width: 1024px;
```

Comment:
don't hardcode width, your catalog container should work fine for any number of cards